ansible/tasks/docker/docker-compose-vara.yaml.j2

```diff
@@ -13,7 +13,7 @@ services:
     volumes:
       - "/home/ec2-user/gear-data/:/gear/"
     command: >
-      gear-node --base-path /gear/ --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0' --prometheus-external
+      gear-node --base-path /gear/ --chain gear/vara_staging.json --telemetry-url 'wss://telemetry.rs/submit 0' --prometheus-external
```

Comment:
is this fine @ukint-vs @sergeyfilyanin ?

Reply:
spec path is ok.